float64 theta
float64 x
float64 s
float64 delta_theta
float64 delta_x
float64 delta_s

float64 xhand_free
float64 zhand_free
float64 theta_free

float64 delta_xhand_free
float64 delta_zhand_free
float64 delta_theta_free

uint8 command_flag
int8 mode
